src/seaport.ts

```diff
@@ -3083,11 +3138,24 @@ export class OpenSeaPort {
     );
 
     const computeOrderParams = () => {
+      const shouldValidate =
+        order.target === merkleValidatorByNetwork[this._networkName];
+
       if ("asset" in order.metadata) {
         const schema = this._getSchema(order.metadata.schema);
         return order.side == OrderSide.Buy
-          ? encodeSell(schema, order.metadata.asset, recipientAddress)
-          : encodeBuy(schema, order.metadata.asset, recipientAddress);
+          ? encodeSell(
+              schema,
+              order.metadata.asset,
+              recipientAddress,
+              shouldValidate ? { validatorAddress: order.target } : undefined
```

Comment:
it seems like we have two ways of stopping the use of merkle validator: not passing in a config, like this, and passing in `isEnglishAuction`. should we just consolidate around the former?